incubator/catalog-inline.spec/src/main/scripts/io/aklivity/zilla/specs/catalog/inline/config/catalog.yaml

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright 2021-2023 Aklivity Inc
+#
+# Licensed under the Aklivity Community License (the "License"); you may not use
+# this file except in compliance with the License.  You may obtain a copy of the
+# License at
+#
+#   https://www.aklivity.io/aklivity-community-license/
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+---
+name: test
+catalogs:
+  catalog0:
+    type: inline
+    options:
+      subjects:
+        subject1:
```

Comment:
Perhaps we can use a sample from avro specification or similar?
If so, is there a logical associated name with the structure represented by the schema?
That would make the example more illustrative.